### PR TITLE
Added all functionalities of the memory card game

### PR DESCRIPTION
**Pull request by aniketkatkar97**
Created the Memory Card Game using HTML, CSS, JavaScript and React.

Game mechanism is as follows - 

1. User will get 10 pairs of cards, arranged randomly, flipped upside down.
2. User will be able to flip 2 cards in a turn.
3. If flipped cards are same, the cards will remain flipped and 5 points will be added to the score.
4. If flipped cards are not same, the card will be flipped back upside down and 1 point will be deducted from the score.
5. Try to flip all the cards in minimum time and turns with maximum score.

Screenshot from 2022-03-22 14-53-50

Screenshot from 2022-03-22 14-52-37

Screenshot from 2022-03-22 14-53-09

Screenshot from 2022-03-22 14-53-42

Screenshot from 2022-03-22 14-52-27

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,209 @@
...
+
+.container h1 {
+  font-family: "Courier New", Courier, monospace;
+  color: green;
+  margin-top: 3vh;
+  margin-bottom: 2vh;
+  font-size: 3em;
+}
+
+#game-info {
```
Comment (maulikkotak98) [inline src/App.css]: use class selector for styling

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,22 @@
 import "./App.css";
+import NavBar from "./components/NavBar";
+import Home from "./components/Home";
+import GameWindow from "./components/GameWindow";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div>
```
Comment (maulikkotak98) [inline src/App.js]: use fragment

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,22 @@
...
+      <Router>
+        <NavBar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/Home" element={<Home />} />
```
Comment (maulikkotak98) [inline src/App.js]: keep path names in lower case

File: src/components/game-window/GameWindow.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+#card-desk {
+  display: grid;
+  padding: auto;
+  justify-content: space-around;
+  align-items: center;
+  grid-template-columns: auto auto auto auto auto;
+  grid-template-rows: auto auto auto auto auto;
+}
+
+#result-panel {
```
Comment (maulikkotak98) [inline src/components/game-window/GameWindow.css]: use class selector

File: src/components/game-window/GameWindow.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+.game-panel {
+  padding: 1em;
+  margin: 0 2em 0 0em;
+  background-color: rgb(247, 252, 243);
+  border: 0.2em rgb(224, 235, 210) solid;
+  box-shadow: inset 0 0 0.6em 0 rgb(47, 87, 47);
+  height: 80%;
+}
+
+#card-desk {
```
Comment (maulikkotak98) [inline src/components/game-window/GameWindow.css]: use class selector

File: src/components/game-window/GameWindow.js [hunk 6]
```diff
@@ -18,6 +19,12 @@ function GameWindow() {
   }, []);
 
   useEffect(() => {
+    compareCoices();
+    // eslint-disable-next-line
```
Comment (maulikkotak98) [inline src/components/game-window/GameWindow.js]: should be //`eslint-disable-next-line react-hooks/exhaustive-deps` to understand why eslint is disabled for that line.

**Comment by maulikkotak98** [comment]
Component folder name should be same as the component name.
Example:
```
GameWindow
 |_ GameWindow.js
 |_ GameWindow.css
```

File: src/App.css (hunk 1 above)
Comment (aniketkatkar97) [reply src/App.css]: Not it is necessary to use id here. That's why I kept it.

File: src/components/game-window/GameWindow.js (hunk 6 above)
Comment (aniketkatkar97) [reply src/components/game-window/GameWindow.js]: Added

**Pull request merged**